recipes/nltk-data/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "nltk" %}
+{%set version = "2017.10.22" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "14b55df9314ece2e7e53258c41b2f29a1fe0d7517ded586fa9ed137edf9a39ec" %}
+
+package:
+  name: {{ name }}-data
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
```

Comment:
This will conflict with the actual `nltk` package, since `name` is `nltk`. Easiest change is `{% set name = "nltk-data" %}` instead at the top, and change the `name` line to just `name: {{ name }}`.

Reply:
done